### DISCUSSION TITLE
Accordion not displayed as the example says

### DISCUSSION THREAD

**Discussion by TimOfTcc** [Megabit/Blazorise#3886] [Q&A] [answered]
I try to use Blazorise 1.0.5 and accordion looks different than the basic example.

![image](https://user-images.githubusercontent.com/55131875/173618852-f6410a32-60ad-41cf-bbac-c53696ee1b12.png)
Could you pls help?

Thanks.

**Comment by stsrki** [comment, marked as answer]
You're probably installed Blazorise.Bootstrap nuget package which is working with bootstrap 4. We are using bootstrap 5 on our web.

**Reply by TimOfTcc** [reply]
Hi,

I just installed nuget pkg as traditional way: Blazorise.Bootstrap5. What further steps I should follow?

**Comment by stsrki** [comment]
You can follow our newest blog https://blazorise.com/blog/how-to-create-a-blazorise-application-beginners-guide

Or read the docs https://blazorise.com/docs/usage/bootstrap5

**Reply by TimOfTcc** [reply]
In .NET 6, Blazor Server project and these section is in Layout.cshtml:
![image](https://user-images.githubusercontent.com/55131875/173816892-ccbcc04c-09ec-4c42-ae90-b85dd0085558.png)

Any ideas pls?

**Reply by stsrki** [reply]
This is something MS recently changed. Just do it in Layout.cshtml instead.

**Reply by stsrki** [reply]
PS. I have updated the docs.
